docs/pages/play.vue

```diff
@@ -80,16 +82,15 @@
           <input type="hidden" name="resources" :value="[...exportData.externalCss, exportData.externalJs].join(',')">
           <input type="hidden" name="css" :value="exportData.css">
           <input type="hidden" name="js_wrap" value="l">
-          <b-btn size="sm" type="submit" :disabled="!isOk">JSFiddle</b-btn>
+          <b-btn size="sm" type="submit" :disabled="!isOk || !ready">JSFiddle</b-btn>
         </b-form>
       </b-col>
     </b-row>
 
-    <!-- Editors -->
+    <!-- Editors / Result / Console -->
     <transition-group
-      v-if="!loading"
       tag="div"
-      class="row"
+      class="row min-vh-100"
```

Comment:
Why was this class added? It moves the footer way too much down and makes no sense to me.

Reply:
Actually it can be removed now... it was there when the editors and results were hidden until ready.